specs/commandline/verify.md

```diff
@@ -4,6 +4,20 @@
 
 Use `notation verify` command to verify signatures on an artifact. Signature verification succeeds if verification succeeds for at least one of the signatures associated with the artifact. The digest of the supplied artifact is returned upon successful verification. It is recommended that this digest reference be used to pull the artifact subsequently, as registry tags may be mutable, and a tag reference can point to a different artifact that what was verified.
 
+Upon successful verifying, the output message is printed out as following:
+
+```text
+Verify succeeded for <registry>/<repository>@<digest>.
+```
+
+If a `tag` is used to identify the OCI artifact, the output message is as following:
+
+```text
+Warning: A tag is used to identify the artifact for verifying. Artifact tags are mutable. Use digests to uniquely identify artifacts and avoid mutability.
+Resolving artifact tag '<tag>' to digest '<digest>' before verifying.
+Verify succeeded for <registry>/<repository>@<digest>.
```

Comment:
```suggestion
```text
Warning: Always verify artifact using digest(`@sha256:...`) rather than a tag(`:latest`) because tags are mutable and a tag reference can point to a different artifact than the one verified.
Successfully verified <registry>/<repository>@<digest>
```

Reply:
Accept other comments but keep "Resolving artifact tag '<tag>' to digest '<digest>' before verifying." since this info is valid if user uses a tag. We still support the `tag`, but resolved to `digest` before verification.